model/src/order.rs

```diff
@@ -309,15 +309,26 @@ impl FromStr for OrderUid {
     }
 }
 
-impl Display for OrderUid {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+impl From<&OrderUid> for String {
+    fn from(uid: &OrderUid) -> Self {
```

Comment:
Docs on `std::string::ToString` say
> This trait is automatically implemented for any type which implements the Display trait. As such, ToString shouldn't be implemented directly: Display should be implemented instead, and you get the ToString implementation for free.

so we should probably do it like that too.

Reply:
Good catch!